script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -781,40 +777,40 @@ It's a perfect name for someone like him, who casts away his body to be born ane
 [sha:ccd3ff8c694b10ede9f9ec21870d9acce23458ef]{
 -- Page 719, Offset 26216.
 -- その響きには薄ら寒いものを感じるが、今までアルクェイドが口にしなかった敵の正体が、ようやく分かった。
-Hearing those words sends chills through my body,
+Hearing those words almost sends chills through my body,
  but at long last, I understand the true nature of the enemy that Arcueid was loathe to speak of.
 }
 [sha:3b3ea0301ab6c95a4b9b9ea88f9e2c243af0307e]{
 -- Page 720, Offset 26217.
 -- 「しかしここからが問題でね。
-"But this is where the real problems start.
+"But here's the real issue.
 }
 [sha:b3a4001231179d2bc4aee1d6f9b8bcec61e7170a]{
 -- Page 720, Offset 26218.
 -- 姫様はもともと吸血鬼を殺すための装置だったんだが、ロアが現れてからはヤツだけを執拗に追いかけるようになっちまった。
-The princess was originally a tool that was used to kill vampires,
- but ever since Roa appeared,
- she's been chasing after him to the exclusion of all else.
+Originally, the Princess was a tool for killin' vampires,
+ but ever since Roa popped up,
+ she's been persistently chasin' after him and him alone.
 }
 [sha:7f4b7f500a7a403de6ade79791a39fd593d50d4b]{
 -- Page 720, Offset 26219.
 -- ロアが今まで転生した回数は十六回。
-To date, Roa has reincarnated sixteen times.
+To date, Roa's reincarnated sixteen times.
 }
 [sha:5110320f4730e432aab807f570805b12efb54a38]{
 -- Page 720, Offset 26220.
 -- そのことごとくを、真祖は消滅させている」
-And the True Ancestor has annihilated him, each and every time."
+And each 'n every time he does, the True Ancestor's exterminates him."
```

Comment:
"exterminated"

Reply:
I think what happened here is I wanted it first to be in past tense, then changed it to present tense but the "'s" (has) is still around even though I meant to get rid of it.